gcloud/datastore/connection.py

```diff
@@ -317,9 +304,9 @@ def commit(self, project, request_pb):
         request_pb.project_id = project
         try:
             return self._stub.Commit(request_pb, GRPC_TIMEOUT_SECONDS)
-        except AbortionError as exc:
-            if exc.code == StatusCode.ABORTED:
-                raise Conflict(exc.details)
+        except _Rendezvous as exc:
```

Comment:
1.  `grpc.Error` --> `grpc.RpcError`
1.  I'm having a hard time with this one. When we catch the error we want to extra information, but `grpc.RpcError` is a [do-nothing subclass][1] of `Exception`:

    ```python
    >>> import grpc
    >>> e = grpc.RpcError()
    >>> e.code()
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
    AttributeError: 'RpcError' object has no attribute 'code'
    >>> e.details()
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
    AttributeError: 'RpcError' object has no attribute 'details'
    ```

    it doesn't (on it's own) have the information we want to extract from a caught exception

[1]: https://github.com/grpc/grpc/blob/df48291ba02a8e79ade846313203593d1feea341/src/python/grpcio/grpc/__init__.py#L253-L254